conda/petsc/run_test.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/bash
-set -e
-
-export PETSC_DIR=${PREFIX}
-
-pkg-config --cflags PETSc | grep -v isystem
```

Comment:
Just double-checking that this is no longer needed. I don't see any changes in the `meta.yaml` regarding testing, so I am assuming we should have removed this a while back?

Reply:
Oh, yeah we can probably put this back. :/ not sure how/why I deleted it.